groups/chd-infrastructure/templates/bep_user_data.tpl

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# Redirect the user-data output to the console logs
+exec > >(tee /var/log/user-data.log|logger -t user-data -s 2>/dev/console) 2>&1
+
+#Create key:value variable
+cat <<EOF >>inputs.json
+${CHD_BACKEND_INPUTS}
+EOF
+#Create cron file and set crontab for CHD user:
+cat <<EOF >>/root/cronfile
+${CHD_CRON_ENTRIES}
```

Comment:
Do you need this, since `bep_cron.tpl` is empty?

Reply:
This will be populated in the future but is not required at this stage.